src/translations/en.json

```diff
@@ -1057,7 +1057,7 @@
             "learn_more": "More information on how to get started.",
             "dialog": {
               "header": "Configure gas consumption",
-              "paragraph": "Gas consumption is the volume of gas that flows from to your home.",
+              "paragraph": "Gas consumption is the volume of gas that flows from the gas grid to your home.",
```

Comment:
I think we should just remove the `from` actually

Reply:
I agree
```suggestion
              "paragraph": "Gas consumption is the volume of gas that flows to your home.",
```